app/routing/VersionedRouting.scala

```diff
@@ -68,9 +69,9 @@ trait VersionedRouting {
             // To avoid a memory leak, we need to ensure we run the request stream and ignore it.
             request.body.to(Sink.ignore).run()
             Future.successful(
-              UnsupportedMediaType(
+              NotAcceptable(
                 Json.toJson(
-                  PresentationError.unsupportedMediaTypeError(
+                  PresentationError.notAcceptableError(
```

Comment:
Similarly, I realise this is out of your changeset, but the lines below (75 to 80), can you delete them, and just return "The Accept header is missing or invalid."?

Reply:
Ideally the scenario is for 406 instead of 413, so corrected it & removed lines 75 to 80 and updated the error desc to above text